SmartDeviceLinkTests/SDLSystemCapabilityManagerSpec.m

```diff
@@ -273,6 +294,29 @@ @interface SDLSystemCapabilityManager ()
             expect(testSystemCapabilityManager.appServicesCapabilities).to(beNil());
         });
     });
+    
+    context(@"when updating display capabilities with OnSystemCapabilityUpdated", ^{
+        __block SDLOnSystemCapabilityUpdated *testUpdateNotification = nil;
+        beforeEach(^{
+            SDLSystemCapability *newCapability = [[SDLSystemCapability alloc] initWithDisplayCapabilities:testDisplayCapabilityList];
+            testUpdateNotification = [[SDLOnSystemCapabilityUpdated alloc] initWithSystemCapability:newCapability];
+            
+        });
+        
+        it(@"should properly update display capability including conversion two times", ^{
+            // two times because capabilities are just saved in first run but merged/updated in subsequent runs
```

Comment:
But you're only posting the same information that was there before, which is an okay test, but you have to also test an update with new data and make sure the merge works correctly.